### ISSUE TITLE
Błąd API: W tej kategorii możesz przyłączyć max 5 ofert do jednego produktu. Znajdź inny produkt w Katalogu lub dodaj nowy.

### ISSUE THREAD

**Issue by kamnord** [allegro/allegro-api#9590]
Dzień dobry,

Nie można w żaden sposób wystawić produktów. Brak możliwości sprawdzenia które oferty są przypisane do produktu.

Produkty, które nie rejestrują się z katalogiem nie są dostępne w katalogu allegro, nie można ich wyszukać po nazwie ani po kodzie EAN.
Przesyłam zdjęcie błędu. 

Nadmienię że korzystam z API przez wtyczkę od twórcy X13

Jeśli jest możliwość wystawiania kolejnych produktów proszę napisać jak pracować z katalogiem, gdzie znajdę produkty wraz z listą przypisanych do nich ofert żeby zweryfikować które 5 ofert jest podpięte pod produkt żeby móc wystawić kolejną ofertę na allegro i podpiąć ofertę do odpowiedniego produktu.
Myślę że jeśli będą przypisane do produktów oferty i będzie ich 5 to będzie można przenieść te błędnie podpięte lub tych ofert których jest za dużo do nowego produktu.

Pozdrawiam
![3](https://github.com/user-attachments/assets/bcd23a28-8e55-4dfd-b13c-1822dc1b95b6)

![1](https://github.com/user-attachments/assets/f347b5db-9a31-4b22-88e7-53f61283224b)

**Comment by AureliuszBrussy** [comment]
Ten komunikat mówi o tym, że dla tego jednego, konkretnego produktu wystawiłeś już 5 ofert. Rozumiem, że twierdzisz, że są wystawione tylko dwie, tak?  Najlepiej, jeżeli wyjaśnisz to przez [formularz kontatkowy](https://allegro.pl/pomoc/kontakt) - wskaż jaką masz trudność i o jaki konkretnie produkt chodzi.

**Comment by kamnord** [comment]
nie przeczytaliście mojej wiadomości. Bardzo proszę ponowne przeczytanie wiadomości.

Ja nie twierdze ile ofert jest przypisanych do produktu - to jest Państwa sugestia ponieważ jak napisałem w pierwszej wiadomości ja nie mam takich informacji jak Pan. Jeśli chodzi o formularz kontaktowy to już przeszedłem tą ścieżkę i zostało stwierdzone że to błąd API i że mam pisać na Githubie, wy macie się zająć tematem.

Proszę prześledzić wczorajszą rozmowę ze strony z formularzami.

Proszę napisać co należ zrobić żeby móc prezentować produkty - tymczasem wychodzi na to że mogę na allegro sprzedawać tylko produkty używane lub to co jest obecnie prezentowane do momentu wygaszenia oferty, najprawdopodobniej po zakończeniu i próbie wznowienia oferty pojawia się błąd co oznacza że nasza działalność na allegro powoli się kończy.

**Comment by AureliuszBrussy** [comment]
Każda oferta prezentuje konkretny produkt z Katalogu Produktów i odpytując o szczegóły oferty można sprawdzić powiązany z ofertą produkt ([GET /sale/product-offers](https://developer.allegro.pl/documentation#operation/getProductOffer)). Możesz posiadać jedną ofertę danego produktu i w razie potrzeby taką ofertę zakańczać/wznawiać. 

Jeżeli miałbym to zweryfikować dokładniej i wskazać id tego produktu, potrzebowałbym trace-id wywołania, które znajduje się w nagłówku response'a z tym komunikatem błędu. 

W kwestii tworzenia produktów - tutaj możesz ew. zwrócić się do supportu integracji, z której korzystasz, ponieważ API Allegro pozwala na utworzenie nowego produktu w Katalogu Produktów  na 2 sposoby. Nie znam szczegółów na temat tego, jak to dokładnie jest zaimplementowane po stronie konkretnej aplikacji.

**Comment by kamnord** [comment]
Niestety nie jestem programistą i w dalszym ciągu nie wiem co zrobić.
Próbuje dodać ofertę i wyświetla się powyższy komunikat zgodnie z przesłanymi zdjęciami. Jak jako użytkownik mam sprawdzić gdzie chce się podpiąć ta oferta i gdzie sprawdzić w allegro w których produktach podpiętych jest 5 ofert i które to są oferty.

Ja jako użytkownik zwyczajnie chciałbym wystawić tą ofertę czy jest taka możliwość ? Jeśli jest to napisz mi krok po kroku co mam zrobić bo siedziałem 3 godziny i wyklikałem wszystkie możliwe opcje. Jak chcesz to możesz sam spróbować żeby ocenić problem. 

Aktualnie nie wiem co mam napisać więc napisze Ci dane produktu który jest zgodny z powyższymi zdjęciami błędu, nie widać go w allegro i nie można go w żaden sposób wystawić jako nowy produkt we wtyczce allegro nie widać powiązania oferty z produktem .
Nazwa: SIŁOWNIK ELEKTRYCZNY 12V SILNIK LINIOWY 600mm 6000N
EAN13: 5904107307602

Ewentualnie możemy wspólnie skonsultować się z twórcą wtyczki która łączy się z allegro - wtyczka rekomendowana przez allegro.

**Comment by AureliuszBrussy** [comment]
@kamnord bez konkretnego przykładu wywołania (trace-id) nie uda mi się sprawdzić, czy mamy tutaj do czynienia z jakimś błędem, lub jaki to jest dokładnie produkt. Dane takie możesz uzyskać od supportu aplikacji. 

Na ten moment wygląda na to, że próbujesz utworzyć ofertę z produktem, dla którego masz już utworzone oferty. Skoro masz już oferty dla tego produktu, wystarczy te oferty aktywować (jeżeli jeszcze nie są aktywne).

**Comment by kamnord** [comment]
Wymieniłem się z korespondencje z twórcą wtyczki integrującej sklep i zasugerował mi żebym wystawił przedmiot na allegro, powinien pojawić się ten sam błąd.
Po przeprowadzonej próbie chciałbym otrzymać wyjaśnienia, w jaki sposób mam wystawić kolejne przedmioty ponieważ robię wszystko zgodnie ze wskazówkami i nie mam takiej możliwości.

Produkt który został zasugerowany w katalogu jest błędny, jedynie co się łączy to model i marka, a co z pozostałymi parametrami? 
Ten produkt różni się od innych: długością, napięciem, siłą, kodem ean, kodem producenta, tytułem, zdjęciem.

W celu weryfikacji załączam zdjęcia z próby wystawienia przedmiotu

![Allegro z1](https://github.com/user-attachments/assets/b51889a3-24a0-439c-8385-2442e1f86c05)
![Allegro z2](https://github.com/user-attachments/assets/006461db-8f3a-406a-8705-313e7ec791b7)
![Allegro z3](https://github.com/user-attachments/assets/13a1f3e3-49db-436a-a7c0-7bce10977b02)

**Comment by AureliuszBrussy** [comment]
Jeżeli chcesz uzyskać pomoc w kwestiach związanych z formularzem wystawiania, skorzystaj z [formularza kontatkowego](https://allegro.pl/pomoc/kontakt). Ja mogę pomóc w kwestiach związanych z API, natomiast potrzebuję trace-id wywołania, które znajduje się w nagłówku naszego response'a z błędem. Trace-id uzyskasz od supportu aplikacji. Support aplikacji może też skontatkować się bezpośrednio z nami i przekazać nam te dane.

**Comment by kamnord** [comment]
Czy to chodzi o poniższe informacje? 
https://allegro.pl/produkt/silownik-elektryczny-24v-silnik-liniowy-450mm-6kn-14b0237e-6e40-40ca-bb3d-92af40bd0211?invoices-b2b=1

cl-b7614-uni-7585-a

**Comment by AureliuszBrussy** [comment]
Nie, to nie jest to. Tak jak wspomniałem, tą informację możesz uzyskać tylko od supportu integracji. Trace-id to identyfikator wywołania. Jeżeli nie jesteś w stanie go uzyskać - skorzystaj z [formularza kontatkowego](https://allegro.pl/pomoc/kontakt) i odnieś się do sytuacji, która wystąpiła w formularzu wystawiania.
